config/printer-creality-cr6se-2020.cfg

```diff
@@ -0,0 +1,119 @@
+# This file contains pin mappings for the stock 2020 Creality CR6-SE.
+# To use this config, during "make menuconfig" select the
+# STM32F103 with a "28KiB bootloader" and with "Use USB for
+# communication" disabled.
+
+# To enable the stock probe, in "make menuconfig" select
+# "Enable extra low-level configuration options" and set "!PA5"
+# for "GPIO pins to set at micro-controller startup".
+
+# Flash this firmware by copying "out/klipper.bin" to a SD card and
+# turning on the printer with the card inserted. The firmware
+# filename must end in ".bin" and must not match the last filename
+# that was flashed.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[stepper_x]
+step_pin: PB8
+dir_pin: !PB7
+enable_pin: !PC3
+rotation_distance: 2.5
+microsteps: 1
+full_steps_per_rotation: 200
```

Comment:
This should be:
```ini
rotation_distance: 40
microsteps: 16
```

Same for all the other steppers.

Reply:
I will change this.